scene/2d/camera_2d.cpp

```diff
@@ -65,15 +65,16 @@ void Camera2D::_update_scroll() {
 
 void Camera2D::_update_process_mode() {
 
-	if (Engine::get_singleton()->is_editor_hint()) {
-		set_process_internal(false);
-		set_physics_process_internal(false);
-	} else if (process_mode == CAMERA2D_PROCESS_IDLE) {
-		set_process_internal(true);
+	// not sure the inside tree is required as process doesn't get sent when outside tree...
```

Comment:
Probably doesn't hurt as this gets called when entering the tree anyway. But yeah, if it's not necessary it could be removed :)

Reply:
Have removed.